cpu/stm32/include/cpu_conf.h

```diff
@@ -115,19 +115,52 @@ extern "C" {
 #define FLASHPAGE_SIZE                  (128U)
 #endif
 
+#ifdef FLASHPAGE_SIZE
 #define FLASHPAGE_NUMOF                 (STM32_FLASHSIZE / FLASHPAGE_SIZE)
+#endif
+
+#if defined(CPU_FAM_STM32F2) || defined(CPU_FAM_STM32F4) || \
+    defined(CPU_FAM_STM32F7)
+#define PERIPH_FLASHPAGE_CUSTOM_PAGESIZES
```

Comment:
Can you add a definition for this under `flashpage.h`

Reply:
Added